src/community/blocks.tsx

```diff
@@ -0,0 +1,347 @@
+
+export const RowBlock: React.FC<RowBlockProps> = ({
+    title,
+    stackingGroup,
+    description,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div
+            style={{
+                display: 'flex',
+                alignItems: 'center',
+            }}
+            aria-label={ariaLabel}
+        >
+            {(title || stackingGroup || description) && (
+                <>
+                    {stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular>{title}</Text2>
+                                </div>
+                            )}
+                            <div style={{zIndex: '0'}}>{stackingGroup}</div>
+                        </div>
+                    )}
+                    {!stackingGroup && (
+                        <div
+                            className={sprinkles({
+                                display: 'flex',
+                                justifyContent: 'space-between',
+                                flex: 1,
+                                alignItems: 'center',
+                            })}
+                        >
+                            {title && (
+                                <div style={{paddingRight: '32px'}}>
+                                    <Text2 regular as="div">
+                                        {title}
+                                    </Text2>
+                                </div>
+                            )}
+                            <div>
+                                <Text2 regular color={vars.colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </div>
+                        </div>
+                    )}
+                </>
+            )}
+        </div>
+    );
+};
+
+interface SimpleBlockProps {
+    image?: RendersNullableElement<typeof Image>;
+    description?: string;
+    'aria-label'?: string;
+}
+
+export const SimpleBlock: React.FC<SimpleBlockProps> = ({image, description, 'aria-label': ariaLabel}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            <div
+                className={sprinkles({
+                    display: 'flex',
+                    alignItems: 'center',
+                })}
+            >
+                {image && (
+                    <div
+                        className={sprinkles({
+                            paddingRight: 16,
+                        })}
+                        style={{zIndex: '0'}}
```

Comment:
the z Index:0 was because it was not possible to click on the images when it was the anchor, but with the current state of the Advanced-Data-Card, this is no longer necessary